cfme/services/catalogs/catalog_item.py

```diff
@@ -140,273 +213,276 @@ def create(self):
         if self.orch_template:
             orch_template = self.orch_template.template_name
 
-        fill(basic_info_form, {'name_text': self.name,
-                               'description_text': self.description,
-                               'display_checkbox': self.display_in,
-                               'select_catalog': catalog.name,
-                               'select_dialog': dialog.name,
-                               'select_orch_template': orch_template,
-                               'select_provider': provider_name,
-                               'select_config_template': self.config_template})
-        if not (self.item_type in provider_required_types):
-            sel.click(basic_info_form.field_entry_point)
+        view.fill({'name': self.name,
+                   'description': self.description,
+                   'display': self.display_in,
+                   'select_catalog': self.catalog.name,
+                   'select_dialog': self.dialog,
+                   'select_orch_template': orch_template,
+                   'select_provider': provider_name,
+                   'select_config_template': self.config_template})
+
+        if view.field_entry_point.value == "":
+            view.fill({'field_entry_point': 'a'})
             if version.current_version() < "5.7":
```

Comment:
**Optional**
Consider moving this to a deferred versionpick of the widget within the view. Or that's crazy talk., I've not tested doing the version picks there. I think it'd be better there since the click_path is the same in both cases its just the widget is different.